### DISCUSSION TITLE
Arch Linux icon isn't working on badges

### DISCUSSION THREAD

**Discussion by hwittenborn** [badges/shields#6597] [Q&A] [answered]
I'm trying to add the Arch Linux logo (code `archlinux`) to a badge, but it doesn't appear to be working.

Any advice?

**Comment by developStorm** [comment, marked as answer]
The code should be `arch-linux` as for now 
`https://img.shields.io/badge/Arch%20Linux-1793D1?logo=arch-linux&logoColor=fff&style=flat-square`
![](https://img.shields.io/badge/Arch%20Linux-1793D1?logo=arch-linux&logoColor=fff&style=flat-square)

But `archlinux` will be working as soon as #6591 is merged.

**Reply by hwittenborn** [reply]
Great, that works. Thanks!

**Reply by developStorm** [reply]
No problem 😆 [Simple Badges](https://github.com/developStorm/simple-badges) might be a good reference if you need more badges with icon ✨

**Reply by PyvesB** [reply]
Thanks for chipping in @developStorm!
